src/main/java/seedu/exercise/model/regime/Regime.java

```diff
@@ -0,0 +1,61 @@
+package seedu.exercise.model.regime;
+
+import seedu.exercise.model.exercise.Exercise;
+import seedu.exercise.model.exercise.UniqueExerciseList;
+
+/**
+ * Represents a Regime in the regime book.
+ */
+public class Regime {
+    private final RegimeName name;
+    private final UniqueExerciseList exercises;
+
+    public Regime(RegimeName name, UniqueExerciseList exercises) {
+        this.name = name;
+        this.exercises = exercises;
+    }
+
+    public void addExercise(Exercise exercise) {
+        exercises.add(exercise);
+    }
+
+    public void deleteExercise(Exercise exercise) {
+        exercises.remove(exercise);
+    }
+
+    public RegimeName getRegimeName() {
+        return name;
+    }
+
+    /**
+     * Returns true if both regimes have the same name.
+     */
+    public boolean isSameRegime(Regime regime) {
+        if (name.equals(regime.getRegimeName())) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
code here can just be devolved into `name.equals(regime.getRegimeName())`. That expression already returns `true/false`

Reply:
ok